test/pkg/environment/aws/expectations.go

```diff
@@ -362,6 +366,41 @@ func (env *Environment) GetAMIBySSMPath(ssmPath string) string {
 	return *parameter.Parameter.Value
 }
 
+// GetSSMPath gets the SSM path for an AMI release specified by the family, k8s version, architecture, and release offset.
+// A release offset of 0 will get you the latest release, 1 the previous release, and so on.
+// Note: this is tighly coupled with the naming conventions for AL2 and AL2023. This is **NOT** compatible with Bottlerocket, Ubuntu, or Windows AMIs.
+func (env *Environment) GetSSMPath(amiFamily string, k8sVersion string, arch string, releaseOffset int) string {
```

Comment:
Since we have cases where offset is just 0, does it make sense to do what we do with K8s version where we have a version that doesn't take in an offset to simplify and then we have an extension of it that takes in an offset when we need it?